modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -31,6 +31,8 @@ If the cluster is configured with an intermediate certificate, add the whole cha
 
 Certificate files are typically stored as Base64 files, with extensions such as `.pem`, `.crt`, `.ca-bundle`, and others. All Secrets that hold certificate files should use the Base64-encoded certificate rather than binary-encoded certificate. The following procedure is applicable for already installed and running instances and for instances that are to be installed.
 
+See also: xref:deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc[].
```

Comment:
This line should go in the Additional resources section.

Reply:
Done